ONSdigital/eq-author-app
# EAR 1425 disable default theme when enabling another theme

### What is the context of this PR?

> Ensured that when a single theme is opened it becomes the preview theme.
> Existing behaviour of when two themes are open with one as preview and you close the previewed theme the other theme becomes the previewed theme is unaffected.
> The previewed theme is not lost if it is closed and navigated away but is updated as in first point if a new theme is selected.

### How to review

> Add to the list below as appropriate, including screenshots when necessary

- Navigate to setting page
- Close the default theme
- Open a new theme and see it now becomes the preview theme
- Reopen theme
- Open a second theme
- Close theme with preview and notice preview moves to second theme

### What to do after everything is green

1. - [ ] Bring this branch up-to-date with Origin/Master
2. - [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. - [ ] Click the **Merge** button on this pull request
4. - [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. - [ ] Move the Jira ticket for this task into the next stage of the process
